web/aos_web/settings.py

```diff
@@ -96,6 +100,14 @@
     }
 }
 
+if IS_TEST:
+    assert DEBUG
+    DATABASES["default"] = {
+        "ENGINE": "django.db.backends.sqlite3",
```

Comment:
Actually I believe sqlite3 comes as a standard Python library (just checked on a clean venv; `import sqlite3` succeeds).

This is a stopgap because setting up postgres during our github builds seemed daunting.  It is something we should do at some point so our tests run against the same DB as deployments.